src/riscv-unpriv-integration.adoc

```diff
@@ -0,0 +1,388 @@
+
+The exact bitwise layout of capabilities as well as the precise encoding of each field for XLEN=32 and XLEN=64 is described in <<app_cap_description>>.
+
+The CHERI model is motivated by the _principle of least privilege_, which
+argues that greater security can be obtained by minimizing the privileges
+accessible to running software. A second guiding principle is the _principle of
+intentional use_, which argues that, where many privileges are available to a
+piece of software, the privilege to use should be explicitly named rather than
+implicitly selected. While CHERI does not prevent the expression of vulnerable
+software designs, it provides strong vulnerability mitigation: attackers have a
+more limited vocabulary for attacks, and should a vulnerability be successfully
+exploited, they gain fewer rights, and have reduced access to further attack
+surfaces.
+
+Protection properties for capabilities include the ISA ensuring that
+capabilities are always derived via valid manipulations of other capabilities
+(_provenance_), that corrupted in-memory capabilities cannot be dereferenced
+(_integrity_), and that rights associated with capabilities shall only ever be
+equal or less permissive (_monotonicity_). Tampering or modifying capabilities
+in an attempt to elevate their rights will yield an invalid capability as the
+tag will be cleared. Attempting to dereference via an invalid capability
+will result in a hardware exception.
+
+CHERI capabilities may be held in registers or in memories, and are loaded,
+stored, and dereferenced using CHERI-aware instructions that expect capability
+operands rather than integer addresses. On hardware reset, initial capabilities
+are made available to software via capability
+registers. All other capabilities will be derived from these initial valid
+capabilities through valid capability transformations.
+
+Developers can use CHERI to build fine-grained spatial and temporal memory
+protection into their system software and applications and significantly
+improve their security.
+
+==== Added State
+
+A CHERI core adds state to allow capabilities to be used from within registers, and to ensure they are not corrupted as they flow through the system.
+This means the following state is added by {cheri_base_ext_name}:
+
+* Extended <<gprs,general purpose integer registers>>: These XLEN-wide registers (e.g. `sp`, `a0`) are all extended with another XLEN bits of capability metadata plus a validity bit.
+When referring to the extended register, the `x` prefix is replaced with a `c`: i.e. the capability extended version of `x0` becomes `c0`.
+The existing integer register names refer to the address part of the capability.
+For the ABI register names the a `c` prefix is added, i.e. `csp`, `ca0`.
+The zero register is extended with zero metadata and a cleared tag: this is called the <<null-cap>> capability.
+
+* Extended system registers: system registers that store addresses are extended to contain capabilities.
```

Comment:
mixing "system registers" and "CSRs"
what's more consistent with the main doc?

Reply:
CSRs is more common. used that